### PR TITLE
fix(ci): fix `dot` unit tests 

### PR DESCRIPTION
**Pull request by timwu20**
- Fixes the `dot` unit tests for the `v0.9.20` branch.
- Removed life runtime support

## Tests

```sh
go test ./dot
```

## Issues

contributes to #2540 

## Primary Reviewer

@edwardmack

### CODE REVIEW

File: dot/utils_test.go [hunk 1]
```diff
@@ -64,7 +63,7 @@ func TestCreateJSONRawFile(t *testing.T) {
 				bs: &BuildSpec{genesis: NewTestGenesis(t)},
 				fp: filepath.Join(t.TempDir(), "/test.json"),
 			},
-			expectedHash: "23356cdb5d3537d39b735726707216c9e329c7b8a2c8a41b25da0f5f936b3caa",
+			expectedHash: "6453fb6118dab594944a7ed68111fdccc3a68253c9d71ac05f27aa2359507c85",
```
Comment (qdm12) [inline dot/utils_test.go:66]: Why did this change? 🤔

File: dot/build_spec_test.go [hunk 2]
```diff
@@ -136,10 +136,10 @@ func TestBuildFromDB(t *testing.T) {
 	}{
 		{name: "normal conditions", path: cfg.Global.BasePath,
 			want: &BuildSpec{genesis: &genesis.Genesis{
-				Name:       "Gossamer",
-				ID:         "gssmr",
+				Name:       "Gossamer Testnet",
+				ID:         "gssmr_test",
 				Bootnodes:  []string{},
-				ProtocolID: "/gossamer/gssmr/0",
+				ProtocolID: "gssmr_test",
```
Comment (qdm12) [inline dot/build_spec_test.go:142]: Why these changes? 🤔

**Review by qdm12 — APPROVED** [review]

File: dot/utils_test.go (hunk 1 above)
Comment (timwu20) [reply dot/utils_test.go:66]: Dan changed the runtime for both `dev` and `gssmr` chains.

File: dot/build_spec_test.go (hunk 2 above)
Comment (timwu20) [reply dot/build_spec_test.go:142]: Dan changed the runtime for both `dev` and `gssmr` chains.
